config/environments/development.rb.sample

```diff
@@ -5,33 +5,64 @@ Rails.application.configure do
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
+  config.autoload = :classic
```

Comment:
This can actually be removed, residue from testing when I was wondering why I couldn't turn off Zeitwerk. Real key was `config.autoloader = :classic` and it's present in application.rb anyway as of now :)